src/core/core.element.js

```diff
@@ -35,12 +35,10 @@ function interpolate(start, view, model, ease) {
 		if (type === typeof origin) {
 			if (type === 'string') {
 				c0 = color(origin);
-				if (c0.valid) {
-					c1 = color(target);
-					if (c1.valid) {
-						view[key] = c1.mix(c0, ease).rgbString();
-						continue;
-					}
+				c1 = color(target);
```

Comment:
Why changing that part of the code? Your changes actually decrease performances because `c0` and `c1` are now always evaluated (`color()`), which is a very costly operation while transitioning values.